src/pages/LoginSignIn/LoginSignInForm.js

```diff
@@ -0,0 +1,223 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import API_URLS from '../../config';
+import ModalLogoLayout from '../CommonComponents/ModalLogoLayout';
+import './LoginSignInForm.scss';
+
+export default class LoginSignInForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      name: '',
+    };
+  }
+
+  requestLogin = e => {
+    e.preventDefault();
+    fetch(API_URLS.SIGNIN, {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.pw,
+      }),
+    })
+      .then(res => {
+        if (res.status === 200) {
+          return res.json();
+        }
+      })
+      .then(res => {
+        if (res) {
+          localStorage.setItem('TOKEN', res.token);
+          this.props.history.push('/review');
+        } else {
+          alert('로그인 하세요');
+        }
+      });
+  };
+
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleDeleteBtn = e => {
+    e.preventDefault();
+    this.setState({
+      [e.currentTarget.previousSibling.name]: '',
+    });
+  };
+
+  checkIdValid = () => {
+    const { id } = this.state;
+    if (id) {
+      const checkIdCondition =
+        /^([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\.([a-z]{2,6}(?:\.[a-z]{2})?)$/i;
+      return checkIdCondition.test(id);
+    } else return true;
+  };
+
+  checkPwValid = () => {
+    const { pw } = this.state;
+    if (pw) {
+      const checkChars = /(?=.*[A-Za-z])/;
+      const checkNums = /(?=.*\d)/;
+      const checkMarks = /(?=.*[$@$!%*#?&])/;
+      const checkCounts = /^[A-Za-z\d$@$!%*#?&]{10,}$/;
+      return (
+        checkCounts.test(pw) &&
+        [checkChars.test(pw), checkNums.test(pw), checkMarks.test(pw)].filter(
+          Boolean
+        ).length >= 2
+      );
+    } else return true;
+  };
+
+  checkNameValid = () => {
+    const { name } = this.state;
+    if (name) {
+      return name.length > 1;
+    } else return true;
+  };
+
+  render() {
+    const { id, pw, name } = this.state;
+    //default : LoginBtn 클릭했을 때 (false)
+    const { isSignBtnClicked } = this.props;
+    const {
+      handleInput,
+      requestLogin,
+      handleDeleteBtn,
+      checkIdValid,
+      checkPwValid,
+      checkNameValid,
+    } = this;
+    const isIdValid = checkIdValid();
+    const isPwValid = checkPwValid();
+    const isNameValid = checkNameValid();
+    const isIdPwBothValid = isIdValid && isPwValid;
+    const isInfoAllValid = isIdValid && isPwValid && isNameValid;
+
+    return (
+      <ModalLogoLayout>
+        {isSignBtnClicked ? (
+          <h2 className="formHeader">회원가입</h2>
+        ) : (
+          <h2 className="formHeader">로그인</h2>
+        )}
```

Comment:
```suggestion
          <h2 className="formHeader">{isSignBtnClicked ? "회원가입" : "로그인" }</h2>
```
단순히 안에 텍스트만 바뀌는 거면 위처럼 처리도 가능해보이네요!